### PR TITLE
Fix: fixes for options

### PR DESCRIPTION
**Pull request by jihochu**
This patch fixes for tflite interpreter option.

Signed-off-by: Jiho Chu <jiho.chu@samsung.com>

### CODE REVIEW

**Review by SeoHyungjun — APPROVED** [review]
LGTM!

**Review by baek2sm — APPROVED** [review]
LGTM

**Review by songgot — APPROVED** [review]
LGTM

**Review by DonghakPark — APPROVED** [review]
LGTM!

File: nntrainer/utils/node_exporter.cpp [hunk 1]
```diff
@@ -24,11 +24,13 @@
 
 namespace {
 
+#ifdef ENABLE_TFLITE_INTERPRETER
```
Comment (jijoongmoon) [inline nntrainer/utils/node_exporter.cpp:27]: nice catch!

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**